python/src/iceberg/io/base.py

```diff
@@ -44,7 +44,7 @@ class InputStream(Protocol):  # pragma: no cover
     def read(self, size: int) -> bytes:
```

Comment:
How about keeping it in line with BinaryIO?
```suggestion
    def read(self, n: int = -1) -> bytes:
```